controllers/users.controllers.js

```diff
@@ -0,0 +1,11 @@
+const { selectUsers } = require("../models/users.models");
+
+exports.getUsers = (req, res, next) => {
+  selectUsers()
+    .then((response) => {
+      res.status(200).send(response);
```

Comment:
Send an object here with a key of users, then in your test you can access body.users rather than just body, better clairty!